src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -367,15 +404,30 @@ export class AppRoot extends mixinBehaviors
           </div>
 
           <!-- Links section -->
-          <paper-listbox on-tap="maybeCloseDrawer">
-            <a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/dataCollection">[[localize('nav-data-collection')]]</a>
+          <paper-listbox>
+            <span on-tap="maybeCloseDrawer"><a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/dataCollection">[[localize('nav-data-collection')]]</a></span>
             <span on-tap="submitFeedbackTapped">[[localize('nav-feedback')]]</span>
-            <a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/">[[localize('nav-help')]]</a>
+            <span on-tap="maybeCloseDrawer"><a href="https://s3.amazonaws.com/outline-vpn/index.html#/en/support/">[[localize('nav-help')]]</a></span>
             <span on-tap="aboutTapped">[[localize('nav-about')]]</span>
-            <div class="legal-links">
-              <a href="https://www.google.com/policies/privacy/">[[localize('nav-privacy')]]</a>
-              <a href="https://s3.amazonaws.com/outline-vpn/static_downloads/Outline-Terms-of-Service.html">[[localize('nav-terms')]]</a>
-              <span on-tap="showLicensesTapped">[[localize('nav-licenses')]]</span>
+            <div id="links-footer">
+              <paper-icon-item id="language-row">
+                <iron-icon id="language-icon" icon="language" slot="item-icon"></iron-icon>
+                <paper-dropdown-menu id="language-dropdown" no-label-float="" vertical-align="bottom">
+                  <paper-listbox slot="dropdown-content" selected="{{language}}" attr-for-selected="value" on-selected-changed="_languageChanged">
+                    <template is="dom-repeat" items="{{supportedLanguages}}" as="lang" sort="_sortLanguageNameAscending">
+                      <paper-item class="language-item" value="{{lang.id}}">
+                        <span class="language-name">{{lang.name}}</span>
+                        <iron-icon icon="check" hidden$="{{_shouldHideLanguageCheckmark(language, lang.id)}}"></iron-icon>
```

Comment:
Is the checkmark working on RTL?

Reply:
Yes!
<img width="912" alt="Screen Shot 2020-06-16 at 1 02 11 PM" src="https://user-images.githubusercontent.com/1009390/84805209-e8fad600-afd1-11ea-9255-651f256352dd.png">